acquia_lift.admin.wizard.inc

```diff
@@ -16,7 +16,10 @@
  */
 function acquia_lift_personalize_campaign_wizard_process_bar_alter(&$form, &$form_state, $form_id) {
   $agent = $form['#agent'];
-
+  $nested = acquia_lift_get_nested_tests($agent);
```

Comment:
From a usability point of view I'm a little wary of this one.  It's weird to users when sometimes they have options and sometimes they don't.  I think an alternative would be to take them to the place where they can confirm their changes upon click.  Of course - I guess that doesn't exist yet.